test_api_crud.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import json
+
+def test_get_user():
+    p = {"page": 2}
+    resp = requests.get("https://reqres.in/api/users", params=p)
+    json_response = resp.json()
+
+    print(json_response["total_pages"])
+    assert json_response["total_pages"] == 2, "total pages doesn't match"
+    print(json_response["total"])
+    assert json_response["total"] == 12, "total doesn't match"
+    print(json_response["data"][0]["email"])
+    assert json_response["data"][0]["email"], "invalid email"
+    print(json_response["support"]["url"])
+    assert (json_response["support"]["url"].endswith("heading")), "url wrong"
+    print(json_response["data"][0]["last_name"])
+    assert json_response["data"][0]["last_name"] != None
+
+def test_get_statuscode():
+    url = "https://reqres.in/api/users"
+    resp = requests.get(url)
+    code = resp.status_code
+    assert code == 200, "Code doesn't match"
+
+def test_create_user():
+    payload = {
+        "name": "Jan",
+        "job": "Automation"
+    }
+
+    resp = requests.post("https://reqres.in/api/users", data=payload)
+    print(resp.json())
+    assert resp.json()["job"] == "Automation", "wrong data!"
+    assert resp.json()["name"] == "Jan", "wrong data!"
+
+    data2 = {
```

Comment:
try to give variable more meaningful names data2 is not especially descriptive, also test should follow AAA convention - Arange Act Assert so keep payloads at the beginning

Reply:
I have applied changes